Source/Fuse.Gestures/Internal/TwoFinger.uno

```diff
@@ -77,17 +75,16 @@ namespace Fuse.Gestures.Internal
 		public event TwoFingerRotateHandler Rotated;
 		public event TwoFingerTranslateHandler Translated;
 		
-		void OnLostCapture()
+		void IGesture.OnLostCapture(bool forced)
 		{
 			_point[0].Down = _point[1].Down = -1;
-			Pointer.ReleaseCapture(this);
 			if (_begun)
 			{
-				_node.EndInteraction(this);
 				if (Ended != null)
 					Ended();
 			}
 			_begun = false;
+			_trackingKeyboard = false;
```

Comment:
I'll reorder. In general I want to call  the event handlers last. I don't know if the entire handling is exception safe though, Gesture as a whole might be an issue, let me check a bit there.